src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -447,6 +471,17 @@ protected function initializeBundles()
     }
 
     /**
+     * The extension point similar to the Bundle::build() method.
```

Comment:
technically, you can already achieve this by overriding ``Kernel::buildContainer``, as long as you take care to start by calling the parent method.